conn_go18.go

```diff
@@ -5,32 +5,27 @@ package clickhouse
 import (
 	"context"
 	"database/sql/driver"
-	"io/ioutil"
-	"net/http"
-	"net/url"
-	"strings"
 )
 
 // Ping implements the driver.Pinger
 func (c *conn) Ping(ctx context.Context) error {
 	if c.transport == nil {
 		return driver.ErrBadConn
 	}
-	// make request with empty body, response should be "Ok"
-	u := &url.URL{Scheme: c.url.Scheme, User: c.url.User, Host: c.url.Host, Path: "/"}
-	req, err := http.NewRequest(http.MethodGet, u.String(), nil)
+
+	req, err := c.buildRequest(ctx, "select version()", nil, true)
 	if err != nil {
 		return err
 	}
-	respBody, err := c.doRequest(ctx, req)
+	_, err = c.doRequest(ctx, req)
```

Comment:
what will return from chproxy if CH is not available ?
I think need to call Close for response to avoid resources leak.